pr-checker/checker.py

```diff
@@ -29,6 +29,7 @@ def adoc_checker(file, valid_tags, rules):
         ":page-releasedate:[ ]*([0-9]{4}[-][0-9]{2}[-][0-9]{2})")
     tags_re = re.compile(":page-tags: *\[(.*)\]")
     list_re = re.compile("^- ")
+    file_tags_re = re.compile("^.*(hide_tags=).*(tags=).*$")
```

Comment:
@priyyanshi @tt-le 
should it be `file_tags_re = re.compile("^.*(tags=).*(hide_tags=).*$")`?